resources/dashboards/Application-Exploration-Dashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+   "hits": 0, 
```

Comment:
I don't know why but when I build the RPM for this PR and install it I do not see this dashboard. 
![screen shot 2018-06-08 at 9 28 52 am](https://user-images.githubusercontent.com/331742/41166835-78cbe17c-6afe-11e8-8929-19c63e1e665f.png)

does it have another dependency that needs to be fulfilled? I've also run in incognito mode just FYI

Reply:
Never mind. **Restarting services was needed**